packages/cadl-ranch-specs/http/type/property/value-types/mockapi.ts

```diff
@@ -91,3 +91,19 @@ Scenarios.Type_Property_ValueTypes_DictionaryString_put = passOnSuccess(dictiona
 const neverMock = createMockApis("never", undefined);
 Scenarios.Type_Property_ValueTypes_Never_get = passOnSuccess(neverMock.get);
 Scenarios.Type_Property_ValueTypes_Never_put = passOnSuccess(neverMock.put);
+
+const unknownStringMock = createMockApis("unknown/string", "hello");
```

Comment:
We have been doing in some other test a single test but takes it all the different values
```
{
 int: 42,
dict: { k1: "hello", k2: 42 }
}
```

not against this method but maybe we should agree one way or another and be consistent